config/forms/2021_2022.json

```diff
@@ -1975,20 +2627,38 @@
                       "value": "Prefer not to say"
                     }
                   }
-                },
+                }
+              },
+              "depends_on": [{
+                "details_known_8": "Yes"
+              }]
+            },
+            "person_8_age": {
+              "header": "",
+              "description": "",
+              "questions": {
                 "age8": {
-                  "check_answer_label": "Person 8’s age",
-                  "header": "What’s person 8’s age",
-                  "hint_text": "",
+                  "header": "Do you know the person 8’s age?",
+                  "check_answer_label": "Person 8’s age",                 
                   "type": "numeric",
                   "min": 0,
-                  "max": 150,
-                  "step": 1
-                },
+                  "max": 120,
+                  "step": 1,
+                  "width": 2
+                }
+              },
+              "depends_on": [{
+                "details_known_8": "Yes"
+              }]
+            },
+            "person_8_sex": {
+              "header": "",
+              "description": "",
+              "questions": {
                 "sex8": {
                   "check_answer_label": "Person 8’s gender",
                   "header": "Which of these best describes person 8’s gender identity?",
-                  "hint_text": "",
+                  "hint_text": "The lead tenant is the person in the household who does the most paid work. If several people do the same paid work, the lead tenant is whoever is the oldest.",
```

Comment:
Is this hint text relevant to Person 8's gender?

Reply:
aah no. it should just be on the lead tenant questions not the additional tenants

Reply:
amended

Reply:
removed